Allow multiple hostname (#2733)
chef/supermarket: docs-chef-io/content/supermarket/config_rb_supermarket.md

`default['supermarket']['fqdn']`

: The fully qualified domain name for the Supermarket server. Defaults to using the current FQDN for the machine.
: For allowing multiple hostname in `default['supermarket']['fqdn']`, specify `node['hostname']` separated by coma.

> Eg. node['https://hostname1.example1.com', https://hostname2.example2.com]

Reviewer: quotes missing for second hostname

> Eg. node['https://hostname1.example1.com', https://hostname2.example2.com]

Reviewer: We should not change anything for the `hostname`. It was only the `allowed_hosts` attribute that needed to be changed. This is a very specific use case which is not so common. So better not to change anything for the `hostname` attribute. We don't know if FQDN can support multiple hostname or not.

> `default['supermarket']['allowed_host']`

Reviewer: ```suggestion
`default['supermarket']['allowed_hosts']`
```

> : This attribute is to set the Allowed Host for supermarket to block arbitrary [Host header injection](https://crashtest-security.com/invalid-host-header/) in the API calls to supermarket. This is by default set as the value of the FQDN(`default['supermarket']['fqdn']`). You can also set this attribute explicitly as the the domain name of your supermarket website e.g. <https://supermarket.chef.io>. You also need to keep the flag: `disable_host_header_attack` as `true` to make this attribute effective. If `disable_host_header_attack` is set to `false` then this attribute will be ignored.

Reviewer: ```suggestion
: This attribute is to set the list of Allowed Hosts for supermarket to block arbitrary [Host header injection](https://crashtest-security.com/invalid-host-header/) in the API calls to supermarket. This is by default set as the value of the FQDN(`default['supermarket']['fqdn']`). You can also set this attribute explicitly as the the domain name of your supermarket website e.g. <https://supermarket.chef.io>. You also need to keep the flag: `disable_host_header_attack` as `true` to make this attribute effective. If `disable_host_header_attack` is set to `false` then this attribute will be ignored.
```

> : For allowing multiple hostname in `default['supermarket']['allowed_host']`,
> specify `node['allowed_hosts']` separated by coma.
> Eg. node['https://hostname1.example1.com', https://hostname2.example2.com]

Reviewer: ```suggestion
: For allowing multiple hostnames in `default['supermarket']['allowed_hosts']`,
specify the values separated by comma e.g. below:
`'https://www.example1.com, https://www.example2.com'`
```
